scicloj/tablecloth.time
# Update README and update github actions

## Goal / Problem

Update the README so it represents more closely what we are doing / hope to do. 

## Solution

A list of changes here:
* Update the version to `1.00-alpha1`. Per our discussion, we will be in `alpha` while we experiment.
* Filled out the README a bit
* Added a github "continuous development" (CD) workflow to test & lint the latest PR. 
* Added a badge  for the status of the CD workflow status
* Added a badge for the latest clojars version (none yet). 

## How to test

Not much to test here. Please read through the README update and check that it's reasonable for the moment. I expect we will update it again very shortly, fillout out the "Usage" section. I think we can make an initial update there after we adjust the library to use the index in TMD.
